src/main/java/Utils/Strings.java

```diff
@@ -0,0 +1,28 @@
+package Utils;
+
+import java.util.Properties;
+
+public class Strings {
```

Comment:
mam wątpliwość co do tej klasy - boję się, że będzie zawierała wiele niepowiązanych ze sobą wartości, co podpowiada mi, że to nie powinno być w jednej klasie